app/controllers/messages_controller.rb

```diff
@@ -40,6 +41,11 @@ def sender
     end
   end
 
+  def unarchive_conversation
+    conversation.update(business_archived_at: nil) if conversation.business_archived_at.present?
+    conversation.update(developer_archived_at: nil) if conversation.developer_archived_at.present?
```

Comment:
We can do both of these at the same time in one `UPDATE`, no need to check the value beforehand.

Reply:
Done ✅

Reply:
Here's the alternatives to those methods:
```
  scope :archived_by_user, -> (dev_or_business) { where("#{dev_or_business} IS NOT NULL").order("#{dev_or_business} DESC") }
  scope :unarchived_by_user, -> (dev_or_business) { where("#{dev_or_business} IS NULL").order("#{dev_or_business} DESC") }
```

Reply:
```
    conversation.update(business_archived_at: nil, developer_archived_at: nil)
```